spring-cloud-gcp-pubsub/src/main/java/com/google/cloud/spring/pubsub/core/subscriber/PubSubSubscriberOperations.java

```diff
@@ -52,12 +52,13 @@
 	 * @return subscriber listening to new messages
 	 * @since 1.1
 	 */
+	@Override
 	Subscriber subscribe(String subscription, Consumer<BasicAcknowledgeablePubsubMessage> messageConsumer);
 
 	/**
 	 * Add a callback method to an existing subscription that receives Pub/Sub messages converted to the requested
 	 * payload type.
-	 * <p>The created {@link Subscriber} is returned so it can be stopped.
+	 * <p>The created {@link SubscriberInterface} is returned so it can be stopped.
```

Comment:
It looks like the return type wasn't changed - did you me to also return a `SubscriberInterface` instead of a `Subscriber`?

If so, this may be a small breaking change since this is in our public API

Reply:
I intentionally tried to avoid this: I'll revert the comment change, but I created a lower level interface and overrode the return type in existing classes. This should not be a breaking change.